app/src/main/java/io/github/droidkaigi/confsched/model/Contributor.java

```diff
@@ -1,16 +1,17 @@
 package io.github.droidkaigi.confsched.model;
 
-import android.support.annotation.Nullable;
+import com.google.gson.annotations.SerializedName;
 
 import com.github.gfx.android.orma.annotation.Column;
 import com.github.gfx.android.orma.annotation.PrimaryKey;
 import com.github.gfx.android.orma.annotation.Table;
-import com.google.gson.annotations.SerializedName;
+
+import android.support.annotation.Nullable;
 
 @Table
 public class Contributor {
 
-    @PrimaryKey
+    @PrimaryKey(auto = false)
```

Comment:
Orma's `@PrimaryKey` is automatically assigned by SQLite, even if `autoincrement = true` is not specified. 

In this case, each ID comes from the API so it is not an "auto" value.

You can see the difference in `Contributor_Schema.java` by comparing both cases: `@PrimaryKey(auto = true)` (default) and `@PrimaryKey(auto = false)`.